src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,7 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        String string = Integer.toBinaryString(value);
```

Comment:
you should create your own implementation. didn't you see the [checklist](https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_02.html)?

Reply:
@Tarabarnik DONE